### DISCUSSION TITLE
Restore codespace status

### DISCUSSION THREAD

**Discussion by Shizanit** [community/community#74837] [Codespaces] [answered]
### Select Topic Area

General

### Body

Can you urgently restore codespace Labs to the state of an hour ago (or earlier), I accidentally deleted the project

**Comment by AidenJamesYt** [comment]
do you have a backup codespace?

**Reply by Shizanit** [reply]
no

**Reply by Shizanit** [reply]
I was just about to make a copy of all the files and got the wrong command.

**Reply by Shizanit** [reply]
or is there a restore codespace button somewhere?

**Reply by AidenJamesYt** [reply, marked as answer]
No, there is no restore button sadly :(

Codespaces are ephemeral environments that are destroyed when you leave them. However, you may be able to recover your project from a backup.

To do this, you will need to contact GitHub support and provide them with the following information:

The name of your codespace
The date and time that you deleted your project
A brief description of what happened

hopefully their support teamwill then be able to investigate your case and see if they can recover it

**Reply by Shizanit** [reply]
create a ticket here https://support.github.com/ticket ?

**Reply by AidenJamesYt** [reply]
yes
